frontend/src/components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,71 @@
+import styles from './LoginForm.module.css';
+import {Button, Form, Input} from "antd";
+import {GoogleOutlined, LockOutlined, UserOutlined} from "@ant-design/icons";
+import React from "react";
+import { Typography } from 'antd';
+const { Title } = Typography;
+
+
+const LoginForm: React.FC = () => {
+    const onFinish = (values: never) => {
+        console.log('Success:', values);
+    };
+
+    const onFinishFailed = (errorInfo: never) => {
+        console.log('Failed:', errorInfo);
+    };
+    return (
+        <div className={styles.loginContainer}>
+            <div className={styles.loginBox}>
+                <Title level={3} className={styles.loginTitle}>Log In</Title>
+                <Form
+                    name="login"
+                    initialValues={{ remember: true }}
+                    onFinish={onFinish}
+                    onFinishFailed={onFinishFailed}
```

Comment:
My Ide has a problem with this line, but it is probably onyl temporary code, so I'm just writing as is.